sdk/typescript/builder/package.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
What are your thoughts on publishing from a dist folder entirely? Basically generating the entire package out into `dist` including these sub-folders and publishing from that directory instead.

Reply:
I was thinking about this, it's definitely an option.  We'd need to add a script to copy over other files, and it get's into some weird issues with how workspaces work which I think might create some big headaches.  I wouldn't love the idea of the linked packages in workspaces working differently than whats published on npm

Reply:
Yeah that's a good point, probably not worth thinking about yet.